### ISSUE TITLE
Dark Mode not working on Language & Locale dropdowns in Personal info

### ISSUE THREAD

**Issue by ArcticFall** [nextcloud/server#29754]
The dropdowns for Language and Locale in the Personal info view have a bright background, making the text illegible.

### Steps to reproduce
1. Enable dark theme under Accessibility
2. Navigate to Personal info
3. Try to select a language or locale from the dropdown

### Expected behaviour
Dropdown should open with a dark background, because the text color is white.

### Actual behaviour
Dropdown opens with a really light gray color making the text illegible.

### Server configuration

**Operating system:** Raspberry Pi OS

**Web server:** Apache Web Server

**Database:** MySQL

**PHP version:** 7.3

**Nextcloud version:** 22.2.3

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated from 21

**Where did you install Nextcloud from:** nextcloud Website

**Comment by ArcticFall** [comment]
The background-color for these elements is set to "inherit", which causes the problem I think.

**Comment by JVT038** [comment]
Yeah, to fix this you need to add a `background-color: var(--color-main-background)` to all `<select>` elements.

**Comment by math-GH** [comment]
Can confirm this bug in Nextcloud 21, 22 ,23 and 24.0.4

![grafik](https://user-images.githubusercontent.com/1645099/185756116-20fa3f74-e1eb-430c-849c-fc16cfa49e61.png)

**Comment by asthana-jee** [comment]
I am new to Open Source. Would like to contribute to it, but couldn't find where to fix the code (as @JVT038 mentioned) :(

**Comment by JVT038** [comment]
I fixed it by installing the [Custom CSS app](https://apps.nextcloud.com/apps/theming_customcss) and adding this code in the theming tab in the settings:
```css
select {
background-color: var(--color-main-background) !important;
}
```

**Comment by asthana-jee** [comment]
Oh alright. So this issue is now solved, right ?

**Comment by JVT038** [comment]
Well, not inherently, because the user still has to tweak CSS to fix it. But it can be fixed manually by the user

**Comment by asthana-jee** [comment]
So shouldn't we add the above code snippet to theming.css ?

**Comment by szaimen** [comment]
Hi, please update to 24.0.9 or better 25.0.3 and report back if it fixes the issue. Thank you!

My goal is to add a label like e.g. 25-feedback to this ticket of an up-to-date major Nextcloud version where the bug could be reproduced. However this is not going to work without your help. So thanks for all your effort!

If you don't manage to reproduce the issue in time and the issue gets closed but you can reproduce the issue afterwards, feel free to create a new bug report with up-to-date information by following this link: https://github.com/nextcloud/server/issues/new?assignees=&labels=bug%2C0.+Needs+triage&template=BUG_REPORT.yml&title=%5BBug%5D%3A+

**Comment by math-GH** [comment]
It is still an issue in 24.0.9

**Comment by math-GH** [comment]
Nice, it is solved in 25.0.3
![grafik](https://user-images.githubusercontent.com/1645099/214111589-c217404e-9859-4dcd-a5ba-7bb40af6d2e6.png)
